mcp/include/command_struct.h

```diff
@@ -294,6 +294,64 @@ typedef struct {
   int update_pbob;
 } lj_pbob_t;
 
+typedef struct {
+  int send_commands;
+  double logOdds; // significance of point sources
+  int update_logOdds; // is this a new commanded value?
+  double latitude; // payload lat
+  int update_lat; // is this a new commanded value?
+  double longitude; // payload long
+  int update_lon; // is this a new commanded value?
+  double heightWGS84; // payload alt above reference surface
+  int update_height; // is this a new commanded value?
+  double exposureTime; // milliseconds
+  int update_exposureTime; // is this a new commanded value?
+  double solveTimeLimit; // time allowed to solve an image
+  int update_solveTimeLimit; // is this a new commanded value?
+  float focusPos; // desired focus position, encoder units
```

Comment:
Should this really be a float? Encoder measurements shouldn't have non-integer units. I would have thought it would have the same units/type as `startPos` and `endPos` (which I would rename like `startFocusPos` or something).

Reply:
We match types to those defined and used in the star camera code. Retyping and matching that is well outside the scope of this module.